app/controllers/users_controller.rb

```diff
@@ -0,0 +1,14 @@
+class UsersController < ApplicationController
+  def leveling
+    @monsters = Monster.all.order(:level)
+  end
+
+  def fight_monster
+    @monster = Monster.find_by(id: params[:monster_id])
+  end
+
+  def example_workspaces
+    msg = NewUserWorkspacesService.new(current_user).call
+    redirect_to root_path, flash: { alert: msg }
```

Comment:
Monad are mainly to direct flow basing on result - Success or Failure. So most often we do something like
```
result = NewUserWorkspacesService.new(current_user).call
result.success? ? something : something else
```